packages/dnslookup/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ameshkov/dnslookup
+TERMUX_PKG_DESCRIPTION="Simple command line utility to make DNS lookups. Supports all known DNS protocols: plain DNS, DoH, DoT, DoQ, DNSCrypt."
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="kay9925@outlook.com"
+TERMUX_PKG_VERSION="1.8.1"
+TERMUX_PKG_SRCURL="https://github.com/ameshkov/dnslookup/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=51b9cbc626e091eb7a98bc326ad026f36d95f8b5917f71f13011466fcdddb3f9
+TERMUX_PKG_BUILD_IN_SRC=true
+
+termux_step_make() {
+	termux_setup_golang
+  
+	# This quic-go version cannot be built on go1.20
+	go get -u && go mod tidy
+
+	export CGO_ENABLED=0
```

Comment:
Please explain why this is necessary. Note that this is known to break some functionality on Android.

Reply:
The system resolver is not used when cgo is disabled on Android. For now patch to fix hardcoded path to `resolv.conf` is not applied to the older version of Go, and this can break some functionality. I don't know if it breaks this specific program, because this is a DNS resolver itself and may not rely on `resolv.conf`.

Reply:
Please test the artifacts anyways. I don't know if it is working.

Reply:
I tested on an aarch64 device and it works fine.